src/rng.cpp

```diff
@@ -1,19 +1,30 @@
-#include "output.h"
 #include "rng.h"
-#include <stdlib.h>
+#include <random>
 
-long rng(long val1, long val2)
+namespace {
+std::mt19937 random_generator {std::random_device{}()};
```

Comment:
If you have a meaningful set of features that would enable, we can hash something out, but that would imply an overhaul of large portions of the game, so they'd need to be some damn good features to be worthwhile.